packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -292,6 +297,29 @@ export class VerticalStackedBarChartBase extends React.Component<
           : // eslint-disable-next-line @typescript-eslint/no-explicit-any
             (xBarScale as any)(lineObject[item][i].xItem.xAxisPoint as string) + this._additionalSpace;
         const y2 = yScale(lineObject[item][i].y);
+
+        if (lineBorderWidth > 0) {
+          borderForLines.push(
+            <line
+              key={`${index}-${i}-line`}
+              x1={x1}
+              y1={y1}
+              x2={x2}
+              y2={y2}
+              opacity={shouldHighlight ? 1 : 0.1}
+              strokeWidth={3 + lineBorderWidth * 2}
+              fill="transparent"
+              strokeLinecap="round"
+              stroke={theme!.palette.white}
+              transform={`translate(${xScaleBandwidthTranslate}, 0)`}
+              {...(isLegendSelected &&
+                selectedLegendTitle === item && {
+                  onMouseOver: this._lineHover.bind(this, lineObject[item][i - 1]),
```

Comment:
you dont need mouse over event for border

Reply:
This is done.